WhiteDentalClinic/WhiteDentalClinic.WebUI/Pages/Appointment/Appointments.razor

```diff
@@ -77,4 +89,123 @@ else
             NavManager.NavigateTo(Routes.Index);
         }
     }
-}
\ No newline at end of file
+}
+
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
```

Comment:
This CSS part can be extracted in a separate file. It will be easier to read, understand and maintain.